### DISCUSSION TITLE
How to push git Tag after successfull publishing of build to playstore.

### DISCUSSION THREAD

**Discussion by furqanullah717** [Triple-T/gradle-play-publisher#1059] [Q&A]
hi, I want to create a tag every time a build is successfully published to the play store. I need to get the versionCode from the build and create a Git Tag. is something like this possible if yes, can you guide me in that direction? 

Thanks.

**Comment by SUPERCILEX** [comment]
I actually wrote a library to do something similar, but flipped around: https://github.com/SUPERCILEX/version-orchestrator.

What you're describing is relatively simple: upon a successful CI build, add a git tag and push.

**Reply by furqanullah717** [reply]
thank @SUPERCILEX 
Sorry for the novice question, I am new to this,  I have the Gradle-play-publisher integrated into my app and it uploads the build to the alpha track by auto-incrementing the version code. What I want to do here is that based on what versionCode is used I want to create a tag. So what I am looking for is to obtain the versionCode after the CI build and push the tag. 

Thanks again.

**Reply by SUPERCILEX** [reply]
Oh, that's a pain. You could probably do it all in Gradle, but the simplest and most reliable approach will likely be to get the version code from the APK/bundle itself: https://stackoverflow.com/questions/13469147/get-android-apk-file-versionname-or-versioncode-without-installing-apk OR https://stackoverflow.com/questions/60155905/how-do-i-get-the-versioncode-and-versionname-badging-of-an-android-aab-file
